python/MODULE.bazel

```diff
@@ -1,102 +1,8 @@
-module(name = "main", version="")
+module(name = "bazel-experiments", version="")
```

Comment:
Curious: what renamed the module? Is it the migration tool or was it manual.

It's inconsequential here, but I wonder.

Reply:
I did it manually. :) 
Similarly, I also removed placeholders (e.g. `# -- use_repo_rule statements -- #`) and a few unnecessary files. Cleanup is needed after running the migration script, see https://bazel.build/external/migration_tool#post-migration-cleanup.